client/dashboard/components/section-header/index.tsx

```diff
@@ -53,7 +54,9 @@ export const SectionHeader = ( {
 							justify="flex-start"
 							expanded={ false }
 							alignment="flex-start"
-							className="dashboard-section-header__actions"
+							className={ clsx( 'dashboard-section-header__actions', {
+								'has-action-dropdown': !! hasActionDropdown,
```

Comment:
Feels weird to introduce this to `SectionHeader`. It seems the `actions` should handle it itself